src/routes/RoutePage.jsx

```diff
@@ -16,6 +16,11 @@ function RoutePage() {
         {/* {loading ? ( */}
         <Routes>
           <Route exact path="/" element={<Hero />} />
+          <Route
+            exact
+            path="/category/:categoryNumber/question/:questionNumber"
+            element={<Quiz />}
+          />
         </Routes>
         {/* ) : ( */}
         {/* <div className="flex items-center h-screen capitalize justify-center text-3xl font-semibold">
```

Comment:
remove commented code if no longer useful

Reply:
I will use it in next pr